frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -913,41 +942,232 @@ function ReaderFooter:addToMainMenu(menu_items)
+                                value_min = 1
+                                value_max = 4
+                            else
+                                value = self.settings.progress_style_thick_height or 7
+                                value_min = 5
+                                value_max = 14
+                            end
+                            local SpinWidget = require("ui/widget/spinwidget")
+                            local items = SpinWidget:new{
+                                width = Screen:getWidth() * 0.6,
+                                value = value,
+                                value_min = value_min,
+                                value_step = 1,
+                                value_hold_step = 2,
+                                value_max = value_max,
+                                title_text =  _("Progress bar size"),
+                                callback = function(spin)
+                                    if self.settings.progress_style_thin then
+                                        self.settings.progress_style_thin_height = spin.value
+                                    else
+                                        self.settings.progress_style_thick_height = spin.value
+                                    end
+                                    self:refreshFooter()
+                                    self.ui:handleEvent(Event:new("SetPageBottomMargin", self.view.document.configurable.b_page_margin))
+                                    UIManager:setDirty(nil, "ui")
+                                end
+                            }
+                            UIManager:show(items)
+                        end
+                    },
+                    {
+                        text = _("Show chapter markers"),
+                        checked_func = function()
+                            return self.settings.toc_markers
+                        end,
+                        enabled_func = function()
+                            return not self.settings.progress_style_thin
+                        end,
+                        callback = function()
+                            self.settings.toc_markers = not self.settings.toc_markers
+                            self:setTocMarkers()
+                            self:updateFooter()
+                            UIManager:setDirty(nil, "ui")
+                        end
+                    },
+                },
+            },
+            {
+                text_func = function()
+                    local pos = _("balanced (10)")
+                    if self.settings.progress_margin_width == 0 then
+                        pos = _("no margins (0)")
+                    end
+                    if self.settings.progress_margin then
+                        local margins = self.ui.document:getPageMargins()
+                        pos = T(_("like book margins (%1)"), math.floor((margins.left + margins.right)/2))
```

Comment:
same as book margins?

Reply:
I wasn't as bothered by this one, and the only other idea I had was something like "honor book margins" or "follow" book margins, and I'm not quite convinced that's any better ;).

Reply:
It's okay, but I think it's less clear than it could be. It can be read as any book. (So about 2 cm perhaps? Who knows. ;-) )